app/src/main/java/org/luxoft/sdl_core/BleSupport/BleCentralService.java

```diff
@@ -0,0 +1,40 @@
+package org.luxoft.sdl_core;
+
+import android.app.Service;
+import android.content.Intent;
+import android.os.IBinder;
+
+public class BleCentralService extends Service {
+        public static final String TAG = BleCentralService.class.getSimpleName();
+
+        @Override
+        public void onCreate() {
+            super.onCreate();
+        }
+
+        @Override
+        public void onDestroy() {
+            //TODO: perform disconnect before calling onDestroy
```

Comment:
@ychernysheva please check if BleCentralService throws the memory leak exception after calling onDestroy.
If it throws, then you should create a separate function that can be called before onDestroy
For better-debugging experience, please comment startSDL and stopSDL functions in the main activity so only BLE transport can be checked

Reply:
Done in 70f115a013c2b887c289081e6554d36713216229